tests/test_images.py

```diff
@@ -1,21 +1,25 @@
 import pytest
+import uuid
 from io import BytesIO
 from aiodocker import utils
 from aiodocker.exceptions import DockerError
 
 
+def _random_name():
+    return "aiodocker-" + uuid.uuid4().hex[:7]
+
+
 @pytest.mark.asyncio
 async def test_build_from_remote_file(docker):
     remote = ("https://raw.githubusercontent.com/aio-libs/"
               "aiodocker/master/tests/docker/Dockerfile")
 
-    tag = "image:1.0"
+    tag = "%s:1.0" % _random_name()
```

Comment:
I checked that in the past and apparently the devs at Python don't want to drop the `%` syntax because is handy and small to use. Plus they even added a new one:

```python
you = "barrachri"
print(f"Hello {you}")
```

So.... well I disagree but I really don't mind at all changing it for consistency sake.